spec/helpers/content_helper_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe ContentHelper do
+  include ContentHelper
+
+  describe '#split_tag' do
+    context 'receives a string' do
+      it 'returns an array delimited at the value given' do
+        example_sentence = 'It is cloudy outside today.'
+        split = split_tag(example_sentence, 'cloudy')
+        expect(safe_join(split)).to eq(
+          safe_join(['It is ', 'cloudy', ' outside today.'])
+        )
+      end
+
+      it 'returns the same string if the delimiter is not present' do
+        example_sentence = 'It is cloudy outside today.'
+        split = split_tag(example_sentence, 'not in set')
+        expect(safe_join(split)).to eq(safe_join([example_sentence]))
+      end
+    end
+
+    context 'receives an array' do
+      it 'returns an array delimited at the value given' do
```

Comment:
This test is returning the same thing as the test below. Is that expected?

Reply:
I don't think so

Reply:
fixed

Reply:
I don't see this fixed. Was the change perhaps lost when you squashed the commits?